### ISSUE TITLE
Add support to remove users no longer working at the company

### ISSUE THREAD

**Issue by marcoman** [electric-cloud/EC-Admin#86]
The request below does not require us to look up users in LDAP or AD, but perhaps we can consider a third option to validate each EC user in LDAP and report/remove the user if they are no longer in the directory.

The request:
Ideally I would have one script that would let me get all the current users listed in the system – say one userid per line.  After that I would use an  internal script to get the list of userids that are no longer active at the company.  I would then feed that list of user ids into another script to remove them from EC.

So, in summary ideally I would like two scripts:
-          One that provides a list of all the current EC user ids
-          One that allows me to programmatically remove EC users by userid

**Comment by imago-storm** [comment]
This issue was moved to electric-cloud-community/EC-Admin#19
